### PR TITLE
New version: Prusa3D.PrusaSlicer version 2.7.0

### PR DESCRIPTION
**Pull request by AlbanDurrheimer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/129137)

### CODE REVIEW

**Review by mivsvit — CHANGES_REQUESTED** [review]
Comparing with

https://github.com/microsoft/winget-pkgs/blob/433580be3750ab084b08227c509703103c210311/manifests/p/Prusa3D/PrusaSlicer/2.6.1/Prusa3D.PrusaSlicer.installer.yaml#L8C8-L8C8

looks like `InstallerType` should be `exe` not `inno`

**Comment by AlbanDurrheimer** [comment]
> Comparing with
> 
> https://github.com/microsoft/winget-pkgs/blob/433580be3750ab084b08227c509703103c210311/manifests/p/Prusa3D/PrusaSlicer/2.6.1/Prusa3D.PrusaSlicer.installer.yaml#L8C8-L8C8
> 
> looks like `InstallerType` should be `exe` not `inno`

It was [Winget-Creator](https://github.com/microsoft/winget-create) who put it in `inno`.

File: manifests/p/Prusa3D/PrusaSlicer/2.7.0/Prusa3D.PrusaSlicer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Prusa3D.PrusaSlicer
+PackageVersion: 2.7.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
```
Comment (stephengillie) [inline manifests/p/Prusa3D/PrusaSlicer/2.7.0/Prusa3D.PrusaSlicer.installer.yaml:8]:
```suggestion
InstallerType: exe
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/p/Prusa3D/PrusaSlicer/2.7.0/Prusa3D.PrusaSlicer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /VERYSILENT
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/p/Prusa3D/PrusaSlicer/2.7.0/Prusa3D.PrusaSlicer.installer.yaml:19]:
```suggestion
- Architecture: x64
```

Comment (AlbanDurrheimer) [reply manifests/p/Prusa3D/PrusaSlicer/2.7.0/Prusa3D.PrusaSlicer.installer.yaml]: Thank you

**Review by stephengillie — APPROVED** [review]

**Pull request merged**